acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ValueTransferAcceptanceTest.java

```diff
@@ -31,6 +31,8 @@
 
 public class ValueTransferAcceptanceTest extends AcceptanceTestBase {
 
+  private static final long NO_OF_TRANSACTIONS = 50;
```

Comment:
Can't this bea local variable in the method that uses it?

Reply:
I just don't like magic numbers. This number is used twice in the code and it doesn't really matter what the number is, only that the same number is used. Using a constant solves this problem.